src/routes/api/trips.route.js

```diff
@@ -2,11 +2,8 @@ import express from 'express';
 import { Trips } from '../../controllers';
 import { Authentication } from '../../middlewares';
 import {
-  onewayCheck,
-  onewayValidateInput,
-  multicityCheck,
-  multicityValidateInput,
-  userRequestHistoryValidator
+  // eslint-disable-next-line max-len
+  onewayCheck, onewayValidateInput, returnTripCheck, returnTripValidateInput, multicityCheck, multicityValidateInput, userRequestHistoryValidator
```

Comment:
Excellent work. However, Instead of disabling eslint, maybe you can move some to the next line, so it can be easier to read.